articles/types/jsdoc.md

```diff
@@ -97,7 +99,14 @@ var Cell = /** @class */ (function () {
 }());
 ```
 
-* `@augments` and `@extends` -- in JsDoc for class-like declarations, specifies which class the upcoming construct extends. Note that `@augments` and `@extends` are synonyms.
+* `@augments` and `@extends` &mdash; if the next construct is a class declaration, specifies which class the upcoming declaration extends. Note that `@augments` and `@extends` are synonyms.
+Like the `@class` tag above, these tags are useful with pre-ES6 javacsript, as in the following example:
+
+```ts
+
+```
+
+The following advanced example shows how to use `@augments` to instantiate a type parameter.
```

Comment:
It would help to say something like  "instantiate a generic class from a Typescript file." And I think it would be easier to understand if you just make it `// @Filename: source.ts` and drop the `declare`. That way the only new concept is type parameters, which *most* people have seen from one of the C-languages.